index.js

```diff
@@ -22,79 +24,39 @@ const DEEPPINK = '#f90297';
 const GRAY = '#9d9d9d';
 const SALMON = '#ff9593';
 
-const STAGGER_HEIGHT = 2;
+var stagger_height = 2;
+var audio_enabled = true;
+var orig_alpha = 1;
 
 const ACTIVE_DURATION = 250;
 
-exports.decorateConfig = config => {
-  return Object.assign({}, config, {
-    cursorShape: 'block',
-    termCSS: `
-      ${config.termCSS || ''}
-      .cursor-node.hypercat-active {
-        opacity: 0 !important;
-      };
-    `,
-    css: `
-      ${config.css || ''}
-      .hypercat-overlay {
-        overflow: hidden;
-        display: none;
-        height: 100%;
-      }
-
-      canvas {
-        display: block;
-        height: 100%;
-        overflow: hidden;
-      }
-
-      .hypercat-overlay.hypercat-active {
-        display: block;
-        background-image: url(file://${css_path});
-        background-repeat: repeat;
-        -webkit-animation: starscroll 4s infinite linear
-      }
-
-      @-webkit-keyframes starscroll {
-        from {background-position:0 0;}
-        to {background-position:-1600px 0;}
-      }
-
-      .hypercat-cursor {
-        position: absolute;
-        pointerEvents: none;
-        background: radial-gradient(circle, ${DEEPPINK} 10%, transparent 10%),
-          radial-gradient(circle, ${DEEPPINK} 10%, ${PINK} 10%) 3px 3px;
-        backgroundSize: 6px 6px;
-        borderWidth: 1px;
-        borderColor: black;
-        borderStyle: solid;
-      }
-      .hypercat-asset {
-        display: none;
-        position: absolute;
-        pointerEvents: none;
-      }
-    `
-  })
-};
-
 
 // Share audio across terminal instances.
 let audio;
 let audioTimeout;
 let audioEnabled;
+
 const playAudio = () => {
+
+  //terminal command
   if (!audioEnabled) {
     return;
   }
```

Comment:
Now that `audioEnabled` is a setting, I think we can get rid of the `hyper-cat-toggle-audio` terminal command. That would mean removing the function above and the `exports.middleware` stuff.